app/views/employment/CheckEmploymentDetailsView.scala.html

```diff
@@ -68,7 +68,7 @@
 
         val stillWorkingForEmployer = EmploymentDataForView(messages(s"$msgKeyGeneric.stillWorkingForEmployer"),
             if(employmentDetails.cessationDate.isDefined) {Some(messages("common.no"))} else {
-                if(employmentDetails.cessationDateQuestion.contains(true)){Some(messages("common.yes"))} else {Some(messages("common.no"))}},
+                if(employmentDetails.cessationDateQuestion.contains(true)){Some(messages("common.no"))} else {Some(messages("common.yes"))}},
```

Comment:
I think the yes and no should be the other way around. When I clicked yes on the question form it displays as no on cya page